libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -719,12 +719,12 @@ def get_num_tokens_from_messages(self, messages: List[BaseMessage]) -> int:
         if sys.version_info[1] <= 7:
             return super().get_num_tokens_from_messages(messages)
         model, encoding = self._get_encoding_model()
-        if model.startswith("gpt-3.5-turbo-0301"):
+        if model.startswith("gpt-3.5-turbo-0301") or model.startswith("gpt-35-turbo-0301"):
```

Comment:
How about `model = model.replace(".", "")` before the if-elif sequence (so that both the cases are covered)?

Reply:
Indeed better, did that and added a comment for future reference.